project/src/components/map/map.tsx

```diff
@@ -21,7 +22,11 @@ function createIcon(iconUrl: string) {
   return icon;
 }
 
-function Map({className, city, offers}: MapProps):JSX.Element {
+const defaultIcon = createIcon(DEFAULT_MARKER_URL);
+const currentIcon = createIcon(CURRENT_MARKER_URL);
+
+
+function Map({className, city, offers, selectedOfferId}: MapProps):JSX.Element {
```

Comment:
пробелов не хватает, все скомкано. 
```suggestion
function Map({className, city, offers, selectedOfferId}: MapProps): JSX.Element {
```